hydradx/notebooks/Omnipool/vault/ImpermanentLossAnalysis2.ipynb

```diff
@@ -0,0 +1,226 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import os\n",
+    "import glob\n",
+    "import random\n",
+    "sys.path.append('../../..')\n",
+    "\n",
+    "from model import processing\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "from model import run\n",
+    "from model import plot_utils as pu\n",
+    "from model.amm.omnipool_amm import OmnipoolState, cash_out_omnipool, value_assets, usd_price, lrna_price\n",
+    "from model.amm.agents import Agent\n",
+    "from model.amm.trade_strategies import omnipool_arbitrage, invest_all, price_sensitive_trading\n",
+    "from model.amm.global_state import GlobalState, fluctuate_prices, historical_prices\n",
+    "\n",
+    "# same seed, same parameters = same simulation result\n",
+    "random.seed(42)\n",
+    "# price_list = processing.import_binance_prices(['BTC', 'ETH', 'DOT'], start_date='Jan 1 2023', days = 120)\n",
+    "\n",
+    "assets = {\n",
+    "    'HDX': {'usd price': 0.05, 'weight': 0.7999},\n",
```

Comment:
Same comment as other notebook

Reply:
For this one, I agree, so I changed it. It didn't make a noticeable difference, but I think it's more correct in principle.